src/StudioCore/Assets/Paramdex/AC6/Defs/DecalParam.xml

```diff
@@ -45,7 +45,8 @@
     <Field Def="f32 emissiveValueEnd"></Field>
     <Field Def="f32 emissiveTime"></Field>
     <Field Def="u8 bIntpEnable"></Field>
-    <Field Def="dummy8 pad_01[3]"></Field>
+    <Field Def="u8 unknown_0x59"></Field>
+    <Field Def="dummy8 pad_01[2]"></Field>
```

Comment:
one of these days we'll do due diligence and check it's a byte and not a short the first time. But I'm happy with this for now.